packages/l/libxcb/xmake.lua

```diff
@@ -16,14 +44,138 @@ package("libxcb")
+        end
+
+        if package:config("damage") then
+            package:add("extsources", "apt::libxcb-damage0-dev")
+        end
+
+        if package:config("dpms") then
+            package:add("extsources", "apt::libxcb-dpms0-dev")
+        end
+
+        if package:config("dri2") then
+            package:add("extsources", "apt::libxcb-dri2-0-dev")
+        end
+
+        if package:config("dri3") then
+            package:add("extsources", "apt::libxcb-dri3-dev")
+        end
+
+        if package:config("present") then
+            package:add("extsources", "apt::libxcb-present-dev")
+        end
+
+        if package:config("glx") then
+            package:add("extsources", "apt::libxcb-glx0-dev")
+        end
+
+        if package:config("randr") then
+            package:add("extsources", "apt::libxcb-randr0-dev")
+        end
+
+        if package:config("record") then
+            package:add("extsources", "apt::libxcb-record0-dev")
+        end
+
+        if package:config("render") then
+            package:add("extsources", "apt::libxcb-render0-dev")
+        end
+
+        if package:config("resource") then
+            package:add("extsources", "apt::libxcb-xrm-dev")
+        end
+
+        if package:config("screensaver") then
+            package:add("extsources", "apt::libxcb-screensaver-dev")
+        end
+
+        if package:config("shape") then
+            package:add("extsources", "apt::libxcb-shape0-dev")
+        end
+
+        if package:config("shm") then
+            package:add("extsources", "apt::libxcb-shm0-dev")
+        end
+
+        if package:config("sync") then
+            package:add("extsources", "apt::libxcb-sync0-dev")
+        end
+
+        if package:config("xevie") then
+            package:add("extsources", "apt::libxcb-xevie0-dev")
+        end
+
+        if package:config("xfree86dri") then
+            package:add("extsources", "apt::libxcb-xfreedri0-dev")
+        end
+
+        if package:config("xfixes") then
+            package:add("extsources", "apt::libxcb-xfixes0-dev")
+        end
+
+        if package:config("xinerama") then
+            package:add("extsources", "apt::libxcb-xinerama0-dev")
+        end
+
+        if package:config("xinput") then
+            package:add("extsources", "apt::libxcb-xinput-dev")
+        end
+
+        if package:config("xkb") then
+            package:add("extsources", "apt::libxcb-xkb-dev")
+        end
+
+        if package:config("xprint") then
+            package:add("extsources", "apt::libxcb-xprint0-dev")
+        end
+
+        if package:config("xtest") then
+            package:add("extsources", "apt::libxcb-xtest0-dev")
+        end
+
+        if package:config("xvmc") then
+            package:add("extsources", "apt::libxcb-xv0-dev")
+        end
+
+        if package:config("xv") then
+            package:add("extsources", "apt::libxcb-xvmc0-dev")
+        end
+    end)
+
```

Comment:
please use for-loop to simply it. like it
https://github.com/xmake-io/xmake-repo/blob/a8ee4ffd7ebeee6dbe360a4ccdfc9951dede4897/packages/f/ffmpeg/xmake.lua#L55-L62